plugin-security.policy

```diff
@@ -74,6 +74,10 @@ grant {
 
   //Enable this permission to debug unauthorized de-serialization attempt
   //permission java.io.SerializablePermission "enableSubstitution";
+
+  //SAML policy
+  permission java.util.PropertyPermission "*", "read,write";
```

Comment:
Won't this be duplicate of https://github.com/opensearch-project/security/blob/2.x/plugin-security.policy#L37?

IMO we should either enumerate the properties individual or only have a single entry with "*" and the list of actions that are permissible.

Reply:
Ahhh true did not notice that we have already such permissions.
I will fix here and will open follow PR in the main branch as well.

Reply:
@cwperks Addressed here https://github.com/opensearch-project/security/pull/3690

Reply:
Added https://github.com/opensearch-project/security/pull/3690